exercises/phone-number/canonical-data.json

```diff
@@ -1,6 +1,6 @@
 {
   "exercise": "phone-number",
-  "version": "1.4.0",
+  "version": "1.4.1",
```

Comment:
This isn't quite right.  The changes you are proposing are more substantial.  "1.5.0" should be the new version. [Test Data Versioning](https://github.com/exercism/problem-specifications#test-data-versioning) tries to explain it.